docs/en/administration/upgrade.md

```diff
@@ -47,5 +47,5 @@ helm upgrade juicefs-s3-gateway juicefs-s3-gateway/juicefs-s3-gateway -n kube-sy
 Please refer to the ["Install and compile the client"](../deployment/hadoop_java_sdk.md#install-and-compile-the-client) document to learn how to install the new version of the Hadoop Java SDK, and then follow the ["Deploy the client"](../deployment/hadoop_java_sdk.md#deploy-the-client) steps to redeploy the new version of the client to complete the upgrade.
 
 :::note
-Some components must be restarted before using the new version of the Hadoop Java SDK. For details, please refer to the ["Restart Services"](../deployment/hadoop_java_sdk.md#restart-services) document.
+Some components must be restarted to use the new version of the Hadoop Java SDK. For details, please refer to the ["Restart Services"](../deployment/hadoop_java_sdk.md#restart-services) document.
```

Comment:
I think we can safely replace all "for details, plz refer to the xxx document" with "refer to [doc title](link) for details", short and sweet.